src/OpenApi/test/Integration/snapshots/OpenApiDocumentIntegrationTests.VerifyOpenApiDocument_documentName=responses.verified.txt

```diff
@@ -78,30 +78,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "type": "object",
```

Comment:
This change is a result of new behavior in the underlying schema generator that removes the `anyOf` property for polymorphic types that don't contain any visible properties. In this case, all the properties on `Shape` are internal given that our implementation doesn't fully support polymorphic types yet which is why this schema is currently empty. This work was pending the incoming $ref version.